statsmodels/genmod/generalized_linear_model.py

```diff
@@ -91,10 +91,10 @@ class GLM(base.LikelihoodModel):
         and should be added by the user (models specified using a formula
         include an intercept by default). See `statsmodels.tools.add_constant`.
     family : family class instance
-        The default is Gaussian.  To specify the binomial distribution
-        family = sm.family.Binomial()
+        The default is Gaussian.  To specify the binomial distribution, use 
+        family = sm.families.Binomial(). 
         Each family can take a link instance as an argument.  See
-        statsmodels.family.family for more information.
+        statsmodels.families.family for more information.
```

Comment:
this should also be `sm.families.family`

`import statsmodels` doesn't import these type of modules

Reply:
Thanks for the comment. I found several places where it says `statsmodels.families.family` --- for example in line 1539, 

```python
See statsmodels.families.family and
        statsmodels.families.varfuncs for more information.
```

Are all these should be `sm.families.family`?

Reply:
I find 14 cases of "statsmodel.fam" in the file
good catch
I add reply as main comment